src/components/CardButton.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { ReactNode, useEffect, useState } from 'react'
+import { useCollection } from 'hooks'
+import { useTranslation } from 'react-i18next'
+import { motion } from 'framer-motion'
+import { isCollected } from 'utils'
+import { CardProps } from './Card'
+import tw from 'twin.macro'
+
+const CardButton = ({ cardData }: CardProps) => {
```

Comment:
I wouldn't reuse component prop types like this, you already have the PokemonCard type to help you not repeat yourself

Reply:
react components prop types should always be 1-to-1